deployment/terraform/agent.go

```diff
@@ -9,13 +9,14 @@ import (
 	"strings"
 	"text/template"
 
+	"github.com/mattermost/mattermost-load-test-ng/deployment"
 	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest"
 
 	"github.com/mattermost/mattermost-server/v5/mlog"
 )
 
-func (t *Terraform) generateLoadtestAgentConfig(output *Output) (*loadtest.Config, error) {
+func generateLoadtestAgentConfig(dpConfig *deployment.Config, output *Output) (*loadtest.Config, error) {
```

Comment:
Why can't we use t.Config here?

Reply:
We can, I think I just forgot to revert from the previous implementation. Will amend :+1: